modules/InboundEmail/language/en_us.lang.php

```diff
@@ -166,5 +166,15 @@
     'LBL_SERVICE' => 'Service',
     'LBL_STORED_OPTIONS' => 'Stored Options',
     'LBL_GROUP_ID' => 'Group ID',
+
+    'LBL_CREATE_CONTACT' => 'Create Contact from Email',
+    'LBL_CREATE_CONTACT_HELP' => 'Select to automatically create a contact record in SuiteCRM from incoming emails.',
+    'LBL_CONTACT_DEFAULT_LANGUAGE' => 'Default Contact Language',
+    'LBL_FILL_CONTACT_NAME' => 'Fill Contact\'s Name',
```

Comment:
Does this need to be escaped? Escaped shouldn't be done in language values else it will bring trouble to translators and thing will break eventually!